docs/plugin-description.md

```diff
@@ -55,27 +55,27 @@ insecure_skip_verify: true
 # api.Fallbacker
 ## timer-fallbacker
 ### description
-```this is a timer fallback trigger when forward fails.```
+this is a timer fallback trigger when forward fails.
 ### defaultConfig
-```
+```yaml
```

Comment:
we'd better add a blank line between title and code block

Reply:
@membphis  
not sure about that , but I found it doesn't have blank line here:
https://www.markdownguide.org/extended-syntax/#syntax-highlighting
please have a look. thanks.

Reply:
It's not mandatory in terms of syntax, just personal preference IMO